Socket Server

Shylexx opened the pull request:
Web Socket Server and Python Client for Gestures.

Hand Tracker serves socket server.
Python Client receives socket server and makes requests to raspberry pi (currently on LAN)
Requires Flask and Websockets from PIP for python client/API

AlmasB commented on python/APICalling.py:
```diff
@@ -0,0 +1,25 @@
+
+def get_status(colour):
+    parameter_1 = colour
+    sql_string = "http://192.168.43.65:5000/{p1}".format(p1=parameter_1)
```
Why is this named `sql_string`?

AlmasB commented on python/APICalling.py:
```diff
@@ -0,0 +1,25 @@
+
+def set_status(colour):
+    parameter_1 = colour
+    sql_string = "http://192.168.43.65:5000/colour/{p1}".format(p1=parameter_1)
```
same comment as above

AlmasB commented on python/APICalling.py:
```diff
@@ -0,0 +1,25 @@
+
+def get_status(colour):
+    parameter_1 = colour
+    sql_string = "http://192.168.43.65:5000/{p1}".format(p1=parameter_1)
```
Can we extract the hardcoded ip and port into modifiable variables at the top of the source file

Shylexx replied:
This is done, now using the args the script is run with to get IP and port.
Just asking Emil about the sql_string and will change as soon as I know

AlmasB reviewed (commented):
This looks really good, thanks. Please see comments attached.

Shylexx commented:
I've changed the name over and extracted the IP and port out of the api calling definitions entirely. It uses the CLI args for the IP and the port so it should be easy to setup with ssh as well this way.
